examples/custom-generator-codelab/package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "generator-sample-app",
+  "name": "sample-app",
```

Comment:
This change can't be made, because we need the packages under `examples/` to have unique names. However, I don't think this one is that big of a deal, because soon the codelab should be updated again to instruct users to run the `@blockly/create-package` script instead of cloning blockly-samples. When those instructions are updated, the user will get their own copy of the files but under whatever name they run the script with.

The change to the html file is fine, but the package.json shouldn't change.

Reply:
Done.  I'll be sure to update the new one too since i had it just named "sample-app".

Reply:
you can see that one here #1602

Reply:
Sorry I wasn't clear, but all of the changes to `package.json` should be reverted -- the author name is also set by the script so won't be relevant [soon]